### DISCUSSION TITLE
[Help] extract text form table

### DISCUSSION THREAD

**Discussion by supernovaMK** [jsvine/pdfplumber#1016] [Ask for help with specific PDFs]
![image](https://github.com/jsvine/pdfplumber/assets/115217054/917153b8-afbd-490b-8f77-ab928cff43fa)

If the example is like above, I would like to output the values ​​corresponding to each preset column value. Is there a way to do this?

Output ex)
Column1:Values1
Column2:Values2
Column3:Values3
Column4:Values4

thank you for your library

**Comment by jsvine** [comment]
Hi @supernovaMK, this seems like a task you'd want to accomplish using standard Python code, *after* extracting the table via `pdfplumber`. For your example of a two-row table, you could try this:

```python
table = page.extract_table()
for key, value in zip(*table):
  print(f"{key}:{value}")
```
